### ISSUE TITLE
Disable or remove codeclimate

### ISSUE THREAD

**Issue by chris1984** [fog/fog-vsphere#258]
Code Climate has been failing for a while now which looks bad on pull requests and releases. 

I propose we turn it off for the time being until we get time to refactor parts of the code that it's upset about.

Thoughts @ezr-ondrej @rlazoryshchak @timogoebel

**Comment by timogoebel** [comment]
@timogoebel, FWIW I'm in favor of removing codeclimate.

**Comment by ezr-ondrej** [comment]
Yeah, we don't need red PRs to know the code is not in the best shape.

**Comment by chris1984** [comment]
Thanks guys, I'll open a PR to remove it
